pkg/apis/events/v1alpha1/cloudbuildsource_types.go

```diff
@@ -68,7 +68,7 @@ type CloudBuildSourceSpec struct {
 
 const (
 	// CloudBuildSource CloudEvent type
-	CloudBuildSourceEvent = "com.google.cloud.build.event"
+	CloudBuildSourceEvent = "google.cloud.cloudbuild.build.v1.statusChanged"
```

Comment:
Can we delete such const like CloudBuildSourceEvent and CloudBuildSourceBuildId since all those are contained under schemas/v1?